common/test/acceptance/pages/lms/course_page.py

```diff
@@ -38,3 +38,32 @@ def has_tab(self, tab_name):
         """
         tab_nav = TabNavPage(self.browser)
         return tab_name in tab_nav.tab_names
+
+    def get_element_padding(self, selector):
```

Comment:
@saleem-latif can we move this to `acceptance\tests\helpers` to make it a reusable function either in lms or studio acceptance tests?